climada_petals/hazard/landslide.py

```diff
@@ -126,11 +127,12 @@ def set_ls_hist(self, bbox, input_gdf, res=0.0083333):
 
         Returns
         -------
-        self : Landslide
+        Landslide : Landslide
             instance filled with historic LS hazard
             set for either point hazards or polygons with specified
             surrounding extent.
         """
+        haz = Landslide()
```

Comment:
Why do you initiate the object with `Landslide()` instead of `cls()`?
```suggestion
        haz = cls()
```

Reply:
I'm not sure, but it works equally. I'll change it though!